tinytroupe/utils/llm.py

```
@@ -0,0 +1,243 @@
+from tinytroupe.openai_utils import LLMRequest
+
+from tinytroupe.utils import logger
+from tinytroupe.utils.rendering import break_text_at_length
+
+################################################################################
+# Model input utilities
+################################################################################
+
+def compose_initial_LLM_messages_with_templates(system_template_name:str, user_template_name:str=None, 
+                                                base_module_folder:str=None,
+                                                rendering_configs:dict={}) -> list:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)